bmq-sdk/src/test/java/com/bloomberg/bmq/resultcodes/GenericResultChecks.java

```diff
@@ -20,10 +20,9 @@
 
 import com.bloomberg.bmq.ResultCodes;
 
-public class GenericResultChecks {
+class GenericResultChecks {
```

Comment:
So unlike others, we still need this as a separate class?

Reply:
yeah, it is used by all the tests in the `com.bloomberg.bmq.resultcodes` package.
maybe I need to look at the structure with a fresh mind one day...